scripts/get_prices.js

```diff
@@ -0,0 +1,59 @@
+#! /bin/env node
+
+const http = require('https');
```

Comment:
I'd rather had it in the cloud repo, because it is not necessary to expose it to the public.

Also we can add the script to the CI and upload the result somewhere and we can use the result both for the Console UI and for the landing and for other places too.

personally i would say that my favorite way to do this was the prices API endpoint

Reply:
It should be public, so it could be used from the landing website

Reply:
It is a public information I see no point in hiding it

Reply:
the prices are public, the process how we retrieve them - not necessarily

Reply:
oh, I didn't notice this is a public repo ...